src/index.html

```diff
@@ -8,6 +8,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div>
+      <div class="stars stars--0">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars stars--1">
+        <div class="stars__star stars--N"></div>
```

Comment:
```suggestion
        <div class="stars__star"></div>
```
delete all classes stars--N
Add one of the next extra classes (a modifier) stars--0, stars--1, stars--2 ... stars--5 to each block
The block with stars--N modifier should have exactly N first stars active